core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/OnBoardingViewModel.kt

```diff
@@ -53,14 +54,24 @@ class OnBoardingViewModel(
         }
     }
 
-    fun onClickNext(){
+    fun onClickExistJoining(){
         settings[PREF_TAG] =  true.toString()
         navController.navigate(RedirectViewModel.DEST_NAME, buildMap {
             putFromSavedStateIfPresent(ARG_NEXT)
             putFromSavedStateIfPresent(ARG_OPEN_LINK)
         })
     }
 
+
+    fun onClickIndividual() {
+        navController.navigate(IndividualLearnerViewModel.DEST_NAME, buildMap {
```

Comment:
Instead of using buildmap when this is empty, use emptyMap() (if this is required).